aconfig.go

```diff
@@ -33,6 +33,8 @@ type Config struct {
 	SkipEnvironment bool
 	SkipFlags       bool
 
+	DoNotGenerateTags bool
```

Comment:
If this config is a public API, it would be nice to have some explanation on which "tags" are not going to be generated.

In the test below, DooDoo has no struct tags, but it seems like something is still generated:
```
		"DooDoo::toml":   "DooDoo",
		"DooDoo::flag":   "doo_doo",
```
I think giving comments for exported fields is a good practice.